CoreHtmlToImage/HtmlConverter.cs

```diff
@@ -10,27 +10,63 @@ namespace CoreHtmlToImage
     /// </summary>
     public class HtmlConverter
     {
-        private const string toolFilename = "wkhtmltoimage.exe";
-        private static readonly string directory;
-        private static readonly string toolFilepath;
+        private static string toolFilename = "wkhtmltoimage";
+        private static string directory;
+        private static string toolFilepath;
 
         static HtmlConverter()
         {
             directory = AppContext.BaseDirectory;
-            toolFilepath = Path.Combine(directory, toolFilename);
 
-            if (!File.Exists(toolFilepath))
+            //Check on what platform we are
+            if (System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform(System.Runtime.InteropServices.OSPlatform.Windows))
```

Comment:
Do you mind changing if/else-if/else to switch/case and have a private method for each operating system.